src/extension/module.js

```diff
@@ -1692,7 +1707,7 @@ import sampleRUM from './rum.js';
         // omit docx extension on sharepoint
         ext = '';
       }
-      if (type === 'xlsx' || type === 'spreadsheet') {
+      if (type === 'xlsx' || (type === 'spreadsheet' && ext !== '.xlsx')) {
```

Comment:
Unrelated to the current issue, but was causing a generic error if someone previewed an excel file from gdrive. Now a user-friendly error message gets displayed.